.github/scripts/set-git-user.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -e
+
+# TODO update with opentelemetry-python-bot or a shared otel bot
+git config user.name opentelemetry-java-bot
```

Comment:
Just trying to understand, does [this](https://github.com/open-telemetry/community/issues/809#issuecomment-1016522419) mean it is necessary to create an `opentelemetry-python-bot` Github user?

Reply:
there are three options:
* create an `opentelemetry-python-bot`
* create a shared `opentelemetry-bot`
* use the `opentelemetry-java-bot` for now (I can share a security token for this user privately)

Reply:
fwiw, i would prefer a shared bot that all sigs can use, but it would be good to have this discussion w/ the greater community. this could be a good topic for the maintainers call

Reply:
hey @codeboten! I noticed that the @opentelemetry-bot user has been used in the opentelemetry-collector-contrib repo: https://github.com/open-telemetry/opentelemetry-collector-contrib/issues?q=is%3Aissue+author%3Aopentelemetry-bot+is%3Aclosed, do you happen to know who owns or has access to that? I would love to get that registered at https://github.com/open-telemetry/community/blob/main/assets.md, or we can come up with another bot name if that one is lost / unavailable for sharing.